common/beerocks/btl/include/btl/btl.h

```diff
@@ -50,12 +62,22 @@ class transport_socket_thread : public socket_thread {
     bool bus_connect(const std::string &beerocks_temp_path, const bool local_master);
     void bus_connected(Socket *sd);
 
-    bool send_cmdu_to_bus(ieee1905_1::CmduMessage &cmdu, const std::string &dst_mac,
-                          const std::string &src_mac, uint16_t length);
+    /**
+     * @brief Sends CDMU to transport for dispatching.
+     *
+     * @param cmdu Control Message Data Unit to send.
+     * @param iface_index Network interface index (set to 0 to send on all interfaces).
+     * @param dst_mac Destination MAC address.
+     * @param src_mac Source MAC address.
+     * @param length Message length.
+     * @return True on success and false otherwise.
+     */
+    bool send_cmdu_to_bus(ieee1905_1::CmduMessage &cmdu, uint32_t iface_index,
+                          const std::string &dst_mac, const std::string &src_mac, uint16_t length);
 
 private:
     bool bus_init();
-    bool bus_send(ieee1905_1::CmduMessage &cmdu, const std::string &dst_mac,
+    bool bus_send(ieee1905_1::CmduMessage &cmdu, uint32_t iface_index, const std::string &dst_mac,
```

Comment:
I don't understand it.
The message is sent to the bus with src mac which is the bridge/al_mac.
doesn't ifindex means index of radio interface? Because we send the message from the "bridge" interface. Does it mean face index of the bridge maybe? It is not clear to me.
Anyway, I don't think that it is a good idea to add it as an extra argument. For whatever reason we need it, couldn't it be generated internally on the function?

Reply:
Please ignore all commits except the last one
I made a mistake and created the PR before committing my code. Sorry for that
Your comment belongs to #1420, just in case you want to review it